gitops/base/backend/deployments.yaml

```
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: backend
+  labels:
+    app.kubernetes.io/name: backend
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: backend
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: backend
+    spec:
+      containers:
+        - name: canada-dental-care-plan-backend
+          # Note: image tag should be pinned to a specific version in overlays
+          image: dtsrhpdevscedacr.azurecr.io/canada-dental-care-plan/canada-dental-care-plan-backend:latest
+          ports:
+            - name: http
+              containerPort: 8080
+          #livenessProbe:
+          #  tcpSocket:
+          #    port: http
+          #readinessProbe:
+          #  httpGet:
+          #    path: /api/readyz
+          #    port: http
```

```suggestion
          livenessProbe:
            httpGet:
              path: /actuator/health/liveness
              port: http
            initialDelaySeconds: 30
          readinessProbe:
            httpGet:
              path: /actuator/health/readiness
              port: http
            initialDelaySeconds: 30
```

The API is configured with kubernetes liveness and readiness probes, so you should use them here.

---

done